utils/angular-output-target/angular-component-lib/utils.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable */
 /* tslint:disable */
-import { fromEvent } from 'rxjs';
+/**
+ * Custom modification
+ * - Replaced import { fromEvent } from 'rxjs'; with import { EventEmitter } from '@angular/core';
+ */
```

Comment:
Oops I commented on the generated file first sorry!

```suggestion
/**
 * GCDS Modification
 * Replaced 
 *     import { fromEvent } from 'rxjs'; 
 * From: https://github.com/ionic-team/stencil-ds-output-targets/blob/9524c1ce970770e01afb493c292f71a2fe61b14a/packages/angular-output-target/angular-component-lib/utils.ts#L3
 */
```